tao_compiler/mlir/disc/transforms/disc_gpu_kernel_to_blob.cc

```diff
@@ -305,9 +305,24 @@ class GpuKernelToBlobPass
 
     xla::HloModuleConfig config;
     xla::DebugOptions options = xla::GetDebugOptionsFromFlags();
-    // Make sure we use full precision division operations.
-    // We need this to pass the ut for `tf.FloorDiv`.
-    (*options.mutable_xla_backend_extra_options())["-nvptx-prec-divf32"] = "2";
+    // The default value is false currently because the UT of tf.FloorDiv UT
+    // fails if prec-div is not 2.
+    bool use_fast_math = false;
```

Comment:
I'm wondering if TF XLA enable fast math by default (we probably should follow the same default value as tf and pytorch)